src/main/resources/application.yml

```diff
@@ -43,7 +43,7 @@ spring:
     job:
       enabled: false  # don't run jobs at application startup
     jdbc:
-      initialize-schema: always
+      initialize-schema: never # we manage the spring batch schema with our own migrations
```

Comment:
can we add a test for this in the `validate_db` job?

Reply:
i think this case is covered with existing integration tests. 

these tests were previously failing before i introduced the changes in the test config.